doc/how-to/recover.md

```diff
@@ -0,0 +1,63 @@
+# MicroCloud Cluster Recovery
+
+Each MicroCloud service uses the [Dqlite](https://dqlite.io/) distributed
+database for highly-available storage. While the cluster recovery process is
+similar for each service, this document only covers cluster recovery for the
+`microcloudd` daemon. For cluster recovery procedures for LXD, MicroCeph and
+MicroOVN, see:
+
+- [LXD Cluster Recovery](https://documentation.ubuntu.com/lxd/en/latest/howto/cluster_recover/)
+- [MicroOVN Launchpad Bug](https://bugs.launchpad.net/microovn/+bug/2072377)
+- [MicroCeph Issue](https://github.com/canonical/microceph/issues/380)
+
+If a MicroCloud cluster loses more than a quorum of its database members, then
+database operations will no longer be possible on the entire cluster. If the
+loss of quorum is temporary (e.g. some members temporarily lose power), database
+operations will be restored when the down members come back online.
+
+This document describes how to recover database access if the down members have
+been lost without the possibility of recovery (e.g. disk failure).
+
+## Recovery Procedure
+
+1. In order to perform cluster recovery, **all cluster members must be shut down**:
+```
+sudo snap stop microcloud
+```
```

Comment:
````suggestion
   ```
   sudo snap stop microcloud
   ```
````

This is a global thing - indent everything that should be under a step with three spaces.